src/components/shared/ManagedTree.js

```diff
@@ -121,15 +124,25 @@ class ManagedTree extends Component<Props, State> {
     const { expanded } = this.state;
     // This file is visible, so we highlight it.
     if (expanded.has(this.props.getPath(highlightItems[0]))) {
-      this.focusItem(highlightItems[0]);
-    } else {
-      // Look at folders starting from the top-level until finds a
-      // closed folder and highlights this folder
-      const index = highlightItems
-        .reverse()
-        .findIndex(item => !expanded.has(this.props.getPath(item)));
-      this.focusItem(highlightItems[index]);
+      return this.focusItem(highlightItems[0]);
+    }
+
+    let index = highlightItems.length - 2;
+
+    if (this.props.autoExpandOnHighlight) {
```

Comment:
I turned `autoExpandOnHighlight` off and it didn't change the fact that the test failed.